portal/views/user.py

```diff
@@ -1359,6 +1359,77 @@ def user_documents(user_id):
     return jsonify(user_documents=[d.as_json() for d in
                                        user.documents])
 
+
+@user_api.route('/user/<int:user_id>/user_documents/<int:doc_id>/download')
```

Comment:
@pbugni In the last web app I worked on, a GET endpoint like '/user_documents/' would return the xml/json of the full list of objects, while '/user_documents/<doc_id>' would return the xml/json of just the specific object in question. For a specific action on that file (eg downloading the full filedata), we would use a more specified path.
However, I don't know if that's standard API design or just was the preference of that design/dev team. If it's more in line with the rest of our API approach, I'm fine with modifying this to remove the /download portion.